### ISSUE TITLE
Block bomb drop

### ISSUE THREAD

**Issue by GoRiLliAz** [rehlds/ReGameDLL_CS#248]
Hi,
Its possible to add this feature ?

Prevent the players from droping the bomb

**Comment by Phantomsq** [comment]
Can be done via amxx. There is no need to add it in gamedll

**Comment by GoRiLliAz** [comment]
Yes by a plugins that I already have elsewhere. But why not just integrated it

**Comment by In-line** [comment]
This has a very low priority, because of already mentioned reason.

**Comment by Nord1cWarr1or** [comment]
This is already achievable — `CBasePlayer::DropPlayerItem` is a hookchain (see `API/CAPI_Impl.h:288`, ReAPI `RG_CBasePlayer_DropPlayerItem`), so a plugin can intercept the `drop` command and block dropping the C4 (`weapon_c4`) selectively, as the maintainers noted. Closing as completed.
